oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -574,7 +537,7 @@ impl Drop for MetricsRecorder {
 
 struct GrpcResponseIterator {
     runtime: RuntimeProxy,
-    response_reader: oak_abi::Handle,
+    response_reader: ReadHandle,
```

Comment:
Should this field be of type `Receiver`? Or should it be renamed to `_handle`?

Reply:
If I move back to waiting on channels and reading raw messages I will change this back to being a raw Handle and rename the field.

Reply:
Changed it to an `Option<Receiver>` (it must be an `Option` to support consuming `receiver` when calling `close` in `drop`)